### PR TITLE
Create bot_config.yml

### PR DESCRIPTION
**Pull request by rthadur**

### CODE REVIEW

File: .github/bot_config.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    # See the License for the specific language governing permissions and
+    # limitations under the License.
+    # ============================================================================
+    #
+    # THIS IS A GENERATED DOCKERFILE.
+    #
+    # This file was assembled from multiple pieces, whose use is documented
+    # throughout. Please refer to the TensorFlow dockerfiles documentation
+    # for more information.
```
Comment (mihaimaruseac) [inline .github/bot_config.yml:20]: all these lines should be unindented

Comment (rthadur) [reply .github/bot_config.yml:20]: done

File: .github/bot_config.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
```
Comment (mihaimaruseac) [inline .github/bot_config.yml]: The other way, all these comments should start at column 1

Comment (rthadur) [reply .github/bot_config.yml]: sorry , it is done now.

**Review by mihaimaruseac — DISMISSED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Pull request closed without merging**